client/src/app/components/environments/environment-list/environment-list-view.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="flex items-center justify-between mb-3">
+  <input pInputText id="commit-hash" (input)="onSearch($event)" [value]="searchInput()" type="text" placeholder="Search for installed systems" class="w-1/3" />
+  @if (!hideLinkToList()) {
+    <a [routerLink]="'/environments'" class="p-button p-2 p-button-secondary self-end">Manage environments</a>
```

Comment:
Looks like ‘Manage Environments’ link is forwarded directly to /environments, can you change it to /repo/:repoId/environment/list